File: MLGStore.UI/src/app/models/customer.ts
export interface CustomerDTO {
  id: number;
  name: string;
  surnames: string;
  address: string;
  username: string;
}

export interface CreateCustomerDTO {
  name: string;
  surnames: string;
  address: string;
  username: string;
  password: string;
}

export interface UpdateCustomerDTO {
  name: string;
  surnames: string;
  address: string;
}

File: Store.Entities/Customer.cs
namespace MLGStore.Entities
{
    public class Customer
    {
        public long Id { get; set; }
        public string Name { get; set; }
        public string Surnames { get; set; }
        public string Address { get; set; }
        public string Username { get; set; }
        public string Password { get; set; }

        public List<CustomerArticle> CustomersArticles { get; set; }
    }
}
